warehouse/ingest-json/src/test/resources/config/ingest/tvmaze-edge-definitions.xml

```diff
@@ -34,180 +34,16 @@ http://www.springframework.org/schema/util/spring-util-4.0.xsd">
                                in the graph.
                              -->
                             <bean class="datawave.ingest.mapreduce.handler.edge.define.EdgeNode">
```

Comment:
You have this edge defined twice in this list.  Was that intentional?

Reply:
Yep. You have to define a source and a sink, even if it's from the same field to itself.

Reply:
do we need to do the same under the `group2` property?

Reply:
No, since that's making edges from fields in group1 to fields in group2